.gitignore

```diff
@@ -138,4 +138,7 @@ dmypy.json
 cython_debug/
 
 # VS Code configuration
-.vscode
\ No newline at end of file
+.vscode
+
+# integration yamls
+pystarport/tests/test_expansion/*.yaml
```

Comment:
I think no need to remove the yaml test cases? since we still support them.

Reply:
just to keep [yaml](https://github.com/crypto-com/pystarport/issues/50) as default